client/app/controllers/tableController.js

```diff
@@ -706,25 +706,34 @@ define(['jquery', 'Handsontable', 'table_template', 'filesaver', 'alertify', 'qt
     return data;
   }
 
-  function updateTableWidth(maxWidth) {
+  function updateInstructionWidth(maxWidth) {
 
-    //$('#instructions').css('width', '1153px');
-    $('#instructions').css('max-width', '1153px');
+    var $instructions = $('#instructions');
+    var $container = $('.container');
+    var $card = $('.card.col-md-10.col-md-offset-1');
```

Comment:
This selector is too specific. We might want to switch to a different width or offset at some point.

Reply:
Good point. Also the column/offset parameters are unnecessary. Fixed this but having trouble pushing changes.